src/components/LoginModal/LoginModal.css

```
@@ -31,6 +31,10 @@
   background-image: url('../../images/metamask.svg');
 }
 
+.dcl.option.metamask-mobile .image {
```

we could re-use the line above:
```suggestion
.dcl.option.metamask .image, .dcl.option.metamask-mobile .image {
  background-image: url('../../images/metamask.svg');
}
```